app/views/partials/_primary_nav.html.erb

```diff
@@ -76,7 +76,7 @@
 
       <% if current_user.logged_in? %>
         <span class="navbar-text">
-          Hello, <%= current_user.display_name %>
+          Hello, <%= link_to current_user.display_name, current_user %>
```

Comment:
Sure thing! I'll add an underline.

That does make it a different style than the links on the left, but I think you're right–this is clearer for now. At some point, I think it'd be worth revisiting the navbar styles to try to make something consistent.